dockerfile-image-update/pom.xml

```diff
@@ -79,7 +79,6 @@
                                     <mainClass>com.salesforce.dockerfileimageupdate.CommandLine</mainClass>
                                 </transformer>
                             </transformers>
-                            <finalName>dockerfile-image-update</finalName>
```

Comment:
Don't we need to update Dockerfile?

Reply:
Yeah, done in b7d4c94 but also needed to resolve the fact that the default artifact is shaded so itest no longer needs dependencies on GSON and GitHub API.

Reply:
And what about https://github.com/salesforce/dockerfile-image-update/blob/master/README.md#running-locally ?

Is there a maven copy target you can do to preserve the "nice" final name?